### PR TITLE
Added site_update.md ISSUE_TEMPLATE

### LINKED ISSUE #34: Create GitHub Issue Template for Site Updates
**Current Behavior**
This repo contains a website. This repo should have an issue template for Site Updates in here https://github.com/layer5io/getnighthawk/tree/master/.github/ISSUE_TEMPLATE.

**Desired Behavior**
Copy the [site_update.md](https://github.com/layer5io/meshery.io/tree/master/.github/ISSUE_TEMPLATE) GitHub issue template from the `layer5io/meshery.io` repo to make a new issue template in the `getnighthawk` repo.

### PR DESCRIPTION
**Pull request by aneeshnema**
**Description**

This PR fixes #34 

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by OlushesiToheeb — COMMENTED** [review]
Looks good to me. 💯

**Review by OlushesiToheeb — APPROVED** [review]
Looks good to me. 💯

**Review by bartkalanski — COMMENTED** [review]
Looking good!

**Review by bartkalanski — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/site_update.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+name: Site Update
+about: Suggest an update to the GetNighthawk.dev website
+title: ''
+labels: 'component/ui'
```
Comment (subhamkrai) [inline .github/ISSUE_TEMPLATE/site_update.md:5]:
```suggestion
labels: 'component/UI'
``` @leecalcote looks better?

Comment (subhamkrai) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: @leecalcote looks better?

Comment (leecalcote) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: @subhamkrai thanks for this. Yes, in general, you're right - it does. A challenge here is that the other 44 repos in the `layer5io` org also use the lowercased version. Changing it here may break label searches across the repos.

Comment (subhamkrai) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: Okay then it makes sense to use lowercase.

**Review by subhamkrai — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/site_update.md (hunk 1 above)
Comment (subhamkrai) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: Approved😀

Comment (leecalcote) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: Awesome. Thank you for considering this, @subhamkrai. It is attention to this type of detail that soothes my OCD.

**Review by leecalcote — APPROVED** [review]
Easy-peasy for @aneeshnema. ✔️

File: .github/ISSUE_TEMPLATE/site_update.md (hunk 1 above)
Comment (subhamkrai) [reply .github/ISSUE_TEMPLATE/site_update.md:5]: Thanks

**Pull request merged**